mappings/net/minecraft/world/EntityDetector.mapping

```diff
@@ -0,0 +1,29 @@
+CLASS net/minecraft/unmapped/C_kkiexczn net/minecraft/world/EntityDetector
+	FIELD f_etrsolop NON_CREATIVE_OR_SPECTATOR_PLAYERS Lnet/minecraft/unmapped/C_kkiexczn;
```

Comment:
```suggestion
	FIELD f_etrsolop NON_CREATIVE_NON_SPECTATOR_PLAYERS Lnet/minecraft/unmapped/C_kkiexczn;
```
I feel like that's slightly clearer

Reply:
How's `...NOR_SPECTATOR...`?

Reply:
Is non/nor correct?
It's usually not/nor or neither/nor

Reply:
right, non-non it is